Archive/Imo/Imo1962Q1.lean

```diff
@@ -111,8 +111,8 @@ theorem case_5_digit {c n : ℕ} (h1 : (digits 10 c).length = 5) (h2 : ProblemPr
 /-- `linarith` fails on numbers this large, so this lemma spells out some of the arithmetic
 that normally would be automated.
 -/
-theorem case_more_digits {c n : ℕ} (h1 : (digits 10 c).length ≥ 6) (h2 : ProblemPredicate' c n) :
-    n ≥ 153846 := by
+theorem case_more_digits {c n : ℕ} (h1 : 6 ≤ (digits 10 c).length) (h2 : ProblemPredicate' c n) :
```

Comment:
The comment complains about linarith being bad. But it got a lot better recently. And there's also omega... Can we improve this now?